config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I think govuk has their own way of managing security policy outside of these apps, so I was told this file wasn't necessary

Reply:
According to this commit https://github.com/alphagov/travel-advice-publisher/pull/919/commits/91889bba73554ee3a775605928399700438837a5 I agree we should delete this file rather than commenting it, as there is GovukContentSecurityPolicy that can be configured instead

Reply:
I see this was added to supersede it in travel-advice-publisher: https://github.com/alphagov/travel-advice-publisher/blob/91889bba73554ee3a775605928399700438837a5/config/initializers/csp.rb